MAUI/Pyramid-Charts/Legend.md

```diff
@@ -82,6 +82,56 @@ this.Content = chart;
 
 {% endtabs %}
 
+## Customizing labels
+
+The appearance of the legend label can be customized using the [`LabelStyle`]() property. 
+
+* [`TextColor`]() – used to change the color of the label.
+* [`FontFamily`]() - used to change the font family for the legend label. 
+* [`FontAttributes`]() - used to change the font style for the legend label. 
+* [`FontSize`]() - used to change the font size for the legend label.
+* [`Margin`]() - used to change the margin size of labels.
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+<chart:SfPyramidChart ItemsSource="{Binding Data}" 
```

Comment:
not added the ItemsSource ,XBindingPath, YBindingPath in cartesian , circular chart , but here added this code snippet